contributor-growth/drafts/contributor-growth-framework/1-Motivating.md

```diff
@@ -0,0 +1,107 @@
+
+## Lowering Contribution Barriers  
+
+Motivation to contribute can be gone as quickly as it appeared. To ensure prospective contributors remain motivated, eliminate as much friction as possible. As mentioned above, each additional step represents an additional friction point and a possible drop-off. Make sure the entire contributor experience is intuitive and easy, including:
+   * Running the code locally before opening a PR (e.g. providing a Makefile and test data on top of instructions to build and run it)
+   * Automated testing during PR process
+   * Clear PR submission requirements (e.g. Contributor License Agreement, Digital Certificate of Origin)
+   * Submitting the PR (and be sure to set the expectations right regarding wait time, but more to that below)
+
+### Documentation
+
+Good docs can be a motivator. It makes it easier to use the project but also shows that the project runs efficiently, that there is a group of responsible engineers behind the project, and that it will likely be a well-organized, well-specified experience. 
+Specify what contributors should expect when they open their first PR. Point to other docs that explain what a PR and branch is and what the workflow looks like. Not everyone may be familiar with it. Consider how your workflow differs from standard workflow concepts and emphasizing those differences through documentation. And make sure contributors at all levels have the tools they need.
+While we all know docs are important, the challenge is often prioritization. If growing contributors is a priority, focus on the first time experience.  
+
+Writing tests can also be a contribution barrier. How does your project expect their tests? Document it and be specific.  
+Also, be clear about where to ask questions if they can't find the answer in the docs. 
+While you may have onboarding docs, they may be outdated or incomplete (after all, when was the last time you needed them?). Leverage new contributors'  "beginner's mind" by encouraging them to improve and/or fix them or at least interview them after they submit their first PR. What's weird, broken, or unclear? If something isn't documented, encourage them to go ahead and add it.
+
+### Minimize Steps: Developer Environment Experience 
+
+The first setup experience is key. Make sure that the development requirements are clear. What languages does the project use? Where can a contributor find the tooling that they need to install to be successful during development? Leverage any tool that reduces steps or makes the entire process smoother.  
+
+Use scripts to install tooling or tests and be clear about what is being installed. Ideally, all your contributor needs is one command to set up the environment, maybe two to run tests. 
```

Comment:
I would also encourage people to consider how someone on Windows should build and run the project locally. If it only works on mac/linux, or requires WSL (i.e. linux) to work, let people know.

Reply:
done